app/clients/hedera/hedera.go

```diff
@@ -1,3 +1,58 @@
 package hedera
 
-// TODO hedera client for interacting with hedera nodes
+import (
+	"encoding/json"
+	"fmt"
+	hederasdk "github.com/hashgraph/hedera-sdk-go"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/model/transaction"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type Client struct {
+	mirrorAPIAddress string
+	mirrorClient     *hederasdk.MirrorClient
+	httpClient       *http.Client
+}
+
+func (hedera Client) GetMirror() *hederasdk.MirrorClient {
+	return hedera.mirrorClient
+}
+
+func NewClient(mirrorAPIAddress, mirrorNodeClientAddress string) *Client {
+	mirrorClient, e := hederasdk.NewMirrorClient(mirrorNodeClientAddress)
+	if e != nil {
+		log.Printf("Error: Could not instantiate mirror client - [%s]", e)
+	}
+
+	return &Client{
+		mirrorAPIAddress: mirrorAPIAddress,
+		mirrorClient:     &mirrorClient,
+		httpClient:       &http.Client{},
+	}
+}
+
+func (hedera Client) GetTransactionsByAccountIdAndTimestamp(account hederasdk.AccountID, lastProcessedTimestamp string) (*transaction.Transactions, error) {
```

Comment:
I think the idiomatic go places the error as the first return parameter, and the result as the second

Reply:
Ignore this, I was drunk!